app/views/tools/widgets/forms/_stock_portfolio_backtest.html.erb

```diff
@@ -0,0 +1,38 @@
+<%= form_with method: :get, data: { turbo_frame: frame_id, controller: "stocks-allocator" } do |form| %>
+  <%= render partial: "shared/section_divider", locals: { section_name: "Portfolio" } %>
+
+  <% 10.times do |i| %>
+    <%= render partial: "shared/stocks_allocator", locals: { list: tool.searchable_stocks.to_json, index: i, required: i == 0 } %>
+  <% end %>
+
+  <div class="flex items-center justify-end gap-2">
+    <span class="text-gray-700 text-xs">Total allocation</span>
+
+    <div class="flex items-center gap-[6px] px-[14px] py-2 h-9 bg-green-400/5 border border-gray-200 rounded-md">
+      <span class="text-black text-sm" data-stocks-allocator-target="totalDisplay">0%</span>
+      <%= lucide_icon "align-vertical-distribute-center", class: "h-4 w-4 text-gray-500 cursor-pointer", "data-action": "click->stocks-allocator#distributeEvenly" %>
+    </div>
+  </div>
+
+  <span class="text-red-500 text-xs" data-stocks-allocator-target="warningText"></span>
+
+  <%= render partial: "shared/section_divider", locals: { section_name: "Settings" } %>
+
+  <%= render partial: "shared/search_select", locals: { list: tool.searchable_stocks.to_json, label: "Benchmark", name: "benchmark_stock", required: true, placeholder: "Search for a stock" } %>
+
+  <div data-controller="synchronized-input" class="flex flex-col gap-2">
+    <%= form.unit_field :investment_amount, label: "Investment amount", value: tool.investment_amount, unit_symbol: "$", data: { controller: "autonumeric" } %>
+  </div>
+
+  <div class="flex flex-col gap-2">
+    <%= form.date_field :start_date, label: "Start date", max: Date.today %>
```

Comment:
Probably worth throwing a `required: true` on this one since it raises if not populated

Also, as i was testing this tool locally, I could not get it to spit out any results.  Loaded stocks to the DB, but there might be other required data I'm missing?

https://github.com/user-attachments/assets/78c89115-5e47-4488-afbe-f7b1ab2f59cb

Reply:
@zachgoll good catch! I've added `required: true` to both dates.

> Loaded stocks to the DB, but there might be other required data I'm missing?

Ah, it's because the start and end dates need to be on different months, since we query Synth on a monthly interval to get the stock prices. Pushed this up to make that a bit more clear: https://github.com/maybe-finance/marketing/pull/134/commits/c44535e1430520444c16e570db35a84f6cd4f60f

Although the text is slightly incorrect. They don't need to be a month apart, just on different months. But struggled to find good wording for that.

Reply:
Got it, makes sense!  I think the wording update works!